core/src/test/java/feign/client/AbstractClientTest.java

```diff
@@ -241,6 +254,10 @@ public void testResponseLength() throws Exception {
 
         @RequestLine("PUT")
         String noPutBody();
+
+        @RequestLine("POST /?foo=bar&foo=baz&qux=")
+        @Headers({"Foo: Bar", "Foo: Baz", "Qux: ", "Content-Type: text/plain;charset=utf-8"})
```

Comment:
yep. FeignTestInterface.java has this:

@RequestLine("POST /")
@Headers({"Happy: sad", "Auth-Token: {authToken}"})
void headers(@Param("authToken") String token);


On Sun, Sep 4, 2016 at 1:37 AM, Andrey Stolyarchuk <notifications@github.com
> wrote:

> In core/src/test/java/feign/client/AbstractClientTest.java
> <https://github.com/OpenFeign/feign/pull/453#discussion_r77440307>:
>
> > @@ -241,6 +254,10 @@ public void testResponseLength() throws Exception {
> >
> >          @RequestLine("PUT")
> >          String noPutBody();
> > +
> > +        @RequestLine("POST /?foo=bar&foo=baz&qux=")
> > +        @Headers({"Foo: Bar", "Foo: Baz", "Qux: ", "Content-Type: text/plain;charset=utf-8"})
>
> Could you please help me to find an example?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenFeign/feign/pull/453/files/e66d0bbe188548b8dbe4fa596ac13668cbdae5a0#r77440307>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAD610FXpsW4OKzwzTBGo6IkzdLL6dzmks5qmbBvgaJpZM4Jz7yl>
> .
>